QualityControl/public/common/filters/filterViews.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { filters } from '../../../common/filters/filter.js';
+import { FilterType } from '../../../common/filters/filterTypes.js';
+import { filtersConfig } from './filtersConfig.js';
+import { h } from '/js/src/index.js';
+
+/**
+ * Creates an input element for a specific metadata field;
+ * @param {object} config - The configuration for this particular field
+ * @param {object} filterMap - An object that contains the keys and values of the filters
+ * @param {Function} onInputCallback - A callback function that triggers upon Input
+ * @param {Function} onEnterCallback - A callback function that triggers upon Enter
+ * @param {Function} onChangeCallback - A callback function that triggers upon Change
+ * @returns {undefined}
+ */
+const createFilterElement = (config, filterMap, onInputCallback, onEnterCallback, onChangeCallback) => {
+  const { filterInput, dynamicSelector } = filters;
```

Comment:
Also, this confused me, you should simply import the desired one from the import statement. 
```js
import { filterInput, dynamicSelector } from './filter.js';
```

Of course with the corresponding change in `filter.js` file for export
```js
export {
  filterInput,
  dynamicSelector,
};
```